packages/demo/usecase-demo/src/utils/wagmiConfig.ts

```diff
@@ -14,33 +14,84 @@ if (!import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID) {
 
 export const projectId = import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID!;
 
-const { chains, publicClient } = configureChains(
-  [bellecour],
-  [walletConnectProvider({ projectId }), publicProvider()]
-);
-
+// WalletConnect metadata
 const metadata = {
-  name: 'Web3Modal',
-  description: 'Web3Modal Example',
-  url: 'https://web3modal.com',
-  icons: ['https://avatars.githubusercontent.com/u/37784886'],
+  name: 'Privacy Pass',
+  description: 'Privacy Pass',
+  url: 'https://iex.ec',
```

Comment:
Info to update here:

```suggestion
  name: 'Content Creator demo dApp',
  description: 'Content Creator demo dApp',
  url: 'https://demo.iex.ec/content-creator',
```